docs/using-the-compiler.rst

```diff
@@ -212,6 +212,9 @@ Input Description
             "cse": false,
             // Optimize representation of literal numbers and strings in code.
             "constantOptimizer": false,
+            // Improve allocation of stack slots for variables, can free up stack slots early.
+            // This can only be activated through the details here.
+            "stackAllocation": false,
```

Comment:
Isn't this a yul optimiser option?

Reply:
Yes it might be. It also influences inline assembly.